Tools/autotest/common.py

```diff
@@ -778,12 +803,16 @@ def arm_vehicle(self, timeout=20):
                 return True
         raise AutoTestTimeoutException("Failed to ARM with mavlink")
 
-    def disarm_vehicle(self, timeout=20):
+    def disarm_vehicle(self, timeout=20, force=False):
         """Disarm vehicle with mavlink disarm message."""
         self.progress("Disarm motors with MAVLink cmd")
+        p2 = 0
+        if force:
+            magic_force_disarm_value = 21196
```

Comment:
Do we really need this temporary variable?

Reply:
Will remove.